inst/resources/environment.py

```diff
@@ -59,46 +100,59 @@ def get_conda(conda=None):
     return None.
     :returns: conda string path to conda or None.
     """
-    if os.environ.get('CONDA_PREFIX', None) is None and conda is None:
+    if os.environ.get("CONDA_PREFIX", None) is None and conda is None:
         return None
     else:
-        return conda or os.environ.get('CONDA_EXE', None)
+        return conda or os.environ.get("CONDA_EXE", None)
 
 
-def get_python_version():
+def get_python_version(environment):
+    # type: (Environment) -> str
+    if environment.package_manager == "conda":
+        versions = conda_version_re.findall(environment.contents)
+        if len(versions) > 0:
+            version = versions[0]
+            if version.count(".") == 1:
+                version = version + ".0"
+            return version
+
     v = sys.version_info
     return "%d.%d.%d" % (v[0], v[1], v[2])
 
 
 def get_conda_version(conda):
     try:
-        args = [conda, '-V']
-        proc = subprocess.Popen(args, stdout=subprocess.PIPE, stderr=subprocess.PIPE, universal_newlines=True)
+        args = [conda, "-V"]
+        proc = subprocess.Popen(args, stdout=subprocess.PIPE, stderr=subprocess.PIPE, universal_newlines=True,)
```

Comment:
Is the trailing comma here intentional?

Reply:
No, I'm guess this was multiline formatted (with the comma) and `black` decided to put it on one line.